maker/src/cfd.rs

```diff
@@ -589,6 +589,10 @@ impl<O, T, W> Actor<O, T, W> {
         }
     }
 
+    // TODO: Because of this we need to have two rollover actor
```

Comment:
I think we could actually refactor the old protocol to remove this step as well.
For me the decision if we want to handle `accept` in a different way is independent of the liquidation feature and could be in a separate PR.

Reply:
Note to self: think about why this TODO is still here AND attempt to split this PR into:

- One that removes the accept/reject automation step.
- One that extracts the new crate and introduces the liquidation feature.

Reply:
As promised:

1. https://github.com/itchysats/itchysats/pull/2533 removes the maker's rollover confirmation step.
2. This PR introduces hourly liquidation events.